typescript/src/deposit.ts

```diff
@@ -42,16 +41,20 @@ export interface Deposit {
   blindingFactor: string
 
   /**
-   * Compressed (33 bytes long with 02 or 03 prefix) Bitcoin public key of
-   * the wallet that is meant to receive the deposit.
+   * Public key hash of the wallet that is meant to receive the deposit. Must
+   * be an unprefixed hex string (without 0x prefix).
+   *
+   * You can use `computeHash160` function to get the hash from a plain text public key.
    */
-  walletPublicKey: string
+  walletPubKeyHash: string
```

Comment:
Let's be consistent with naming and avoid abbreviations:
```suggestion
  walletPublicKeyHash: string
```
Remember about changing all references as well

Reply:
@lukasz-zimnoch I would prefer to stick to `PubKeyHash` instead of `PublicKeyHash` because we have that abbreviation in the contracts. For example we have `activeWalletPubKeyHash` in the `BridgeState.sol` or we have `reveal` parameter of the `revealDeposit` method in the `Deposit.sol` which is a structure that contains `walletPubKeyHash` and `refundPubKeyHash`. The inconsistnency betweent the contracts and the ts lib may be inconvinient.

What do you think? Changin this is not a problem at all so please confirm if you still want to do it and I'll do it 😉

Reply:
Actually, we don't have to be super consistent with what we have in the contracts. The mapping is done in one place, i.e.`ethereum.ts` file. We should rather seek to have everything consistent in the library itself. If we have `PubKey` everywhere in the library, that works for me. What I'm trying to say here is to avoid the case where we have both `PubKey` and `PublicKey` in the library.

Reply:
Changed it to `PublicKey`:
fcc0e42e55b0fde1659bac5dd7ac081241f0cdd1
69dfd10cfee4a7b56dae13a76a9600da11ff171f